### PR TITLE
L33T-1275

### PR DESCRIPTION
**Pull request by notsaltydev**

### CODE REVIEW

File: jameda_interface_config.js [hunk 1]
```diff
@@ -212,10 +214,10 @@ var interfaceConfig = {
     DISABLE_JOIN_LEAVE_NOTIFICATIONS: true,
 
     /**
-    * Decides whether the chrome extension banner should be rendered on the landing page and during the meeting.
-    * If this is set to false, the banner will not be rendered at all. If set to true, the check for extension(s)
-    * being already installed is done before rendering.
-    */
+     * Decides whether the chrome extension banner should be rendered on the landing page and during the meeting.
```
Comment (notsaltydev) [inline jameda_interface_config.js:217]: I don't know why this changes are here. In my git history I didn't affect this part.

File: react/features/base/util/uri.js [hunk 2]
```diff
@@ -556,6 +556,20 @@ export function urlObjectToString(o: Object): ?string {
         }
     }
 
+    const { brandLogoPath } = o;
```
Comment (nehatdedinca) [inline react/features/base/util/uri.js:559]: Why did you assign brandLogoPath to o?

File: jameda_interface_config.js (hunk 1 above)
Comment (nehatdedinca) [reply jameda_interface_config.js:217]: As long as the lint passes, its fine!

File: jameda_interface_config.js [hunk 3]
```diff
@@ -1,9 +1,11 @@
 /* eslint-disable no-unused-vars, no-var, max-len */
+const DEFAULT_IMG_HOST = 'https://security.patientus.de';
 
 var interfaceConfig = {
     // TO FIX: this needs to be handled from SASS variables. There are some
     // methods allowing to use variables both in css and js.
     DEFAULT_BACKGROUND: '#474747',
+    DEFAULT_LOGO_URL: `${DEFAULT_IMG_HOST}/assets/img/general/patientus_logo.png`,
```
Comment (nehatdedinca) [inline jameda_interface_config.js:8]: This will always render the default logo (patientus_logo). The ticket is I guess to make the Logo displayed for default (patientus) and also for the White-Labels. Therefore, I guess you have to check the subdomain and then render the logo which will be displayed in Jitsi. You might have something like ``${DEFAULT_IMG_HOST}/assets/img/general/${LOGO_NAME}``.

Comment (notsaltydev) [reply jameda_interface_config.js:8]: Yes, you have right that will be render the default logo but when we provide a value over the external api the logo will be changed (please have a look here: https://github.com/jameda-telemedicine/jitsi-meet/pull/33/files#diff-41c3cb130f8647313139eeb1e4780e36R363). As we discussed we want to provide logo through the iframe then solution with subdomain here is not a proper way.

File: react/features/base/util/uri.js (hunk 2 above)
Comment (notsaltydev) [reply react/features/base/util/uri.js:559]: Here I did `o` object destructuring and assign the value from external api to enhance url.
